docs/_templates/components_requirements.tmpl

```diff
@@ -1,39 +1,83 @@
-This information is not relevant when installing the orchestrator via RPM as is recommended
-in the :ref:`install-server` section.
 
-.. list-table:: Inmanta core and extensions versions
-   :header-rows: 1
+Module compatibility ranges
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   * - .. centered:: Component
-     - .. centered:: Required version
 
-   {% for key, value in data["component_requirements"] | dictsort %}
-   * - .. centered:: {{ key }}
-     - .. centered:: {{ value }}
-   {% endfor %}
+When an Inmanta module integrates tightly with the Inmanta Service Orchestrator, it's important
+to use a version of that module that is compatible with the orchestrator. This can be achieved by either:
 
-.. note::
+- Using a specific index (or git repo) per ISO version and making sure it contains the compatible versions of the modules.
+- Using a generic index (or git repo) possibly containing multiple versions for a given module and adding a
+  project constraint file for each ISO version in which the compatible module ranges are specified.
+
+{% set list_only = ["lsm", "std"] %}
+
+.. only:: iso
+
+    The table below shows the set of Inmanta modules that integrate tightly with version |version_major| of the
+    Inmanta Service Orchestrator and which versions are compatible with this ISO release.
+
+    .. list-table:: Modules
+       :header-rows: 1
+
+       * - Module
+         - Version constraint
+
+       {% for key, value in data["module_compatibility_ranges"]  | dictsort %}
+       {% if key in list_only %}
+       * - {{ key }}
+         - .. centered:: {{ value }}
+       {% endif %}
+       {% endfor %}
+
+.. only:: oss
+
+    The table below shows the set of Inmanta modules that integrate tightly with this Inmanta OSS version |release| and which versions are compatible with this OSS release.
+
+    .. list-table:: Modules
+       :header-rows: 1
+
+       * - .. centered:: Module
 
-    This information is also available as a constraints file in `requirements.components.txt <./requirements.components.txt>`_
-    and in json format in the `compatibility.json <./compatibility.json>`_ file.
+       {% for key, value in data["module_compatibility_ranges"] | dictsort %}
+       {% if key in list_only %}
+       * - {{ key }}
+       {% endif %}
+       {% endfor %}
 
+.. note::
+
+    This information is also available in a machine-consumable format in the `compatibility.json <./compatibility.json>`_ file.
 
-{% else %}
-Python components are not pinned for major version |version_major| because it is not released yet.
 {% endif %}
 
+
+.. only:: iso
+
+    {% if data['component_requirements'] %}
+
+    .. note::
+
+        To transparently install the latest compatible versions for all the python components that make up the
+        ``inmanta-service-orchestrator`` package for version |version_major| of the Inmanta Service Orchestrator,
+        install it via the following command:
```

Comment:
What do you mean with "transparantly". This method is not really transparent right, because all the version filtering happens in the background based on the version constraints embedded in the product package. Do you mean conveniently?
```suggestion
        To conveniently install the latest compatible versions for all the python components that make up version |version_major| of the Inmanta Service Orchestrator product,
        install the product package using the following command:
```

Reply:
> What do you mean with "transparantly"

I meant it in the sense "without the user being aware" since it happens in the background without the need of specifying versions for components

Reply:
I see. I just learned that transparent can also have another meaning than the one I had in mind and that is what you are using it form.

Reply:
I'll go with conveniently if it's clearer (pun intended :) )